__tests__/client.test.js

```diff
@@ -62,6 +62,11 @@ describe("generator-engage:client", () => {
       expect(json.dependencies["webpack-cli"]).toBeDefined();
     });
 
+    it("aliases react-dom as @hot-loader/react-dom in dev dependencies", () => {
+      json = readPackageJson();
+      expect(json.devDependencies["react-dom"]).toEqual("npm:@hot-loader/react-dom");
+    });
+
```

Comment:
@dpickett current behavior has this failing, saying it's inserting it with the dependency of ~16.13 (not the new devDependency), and the test on line 38 also failing, saying it's not inserting it in dependencies at all. Was having trouble getting console.logs to show up, and debuggers aren't allowing the test suite to run.

Reply:
this is not the dev dependency. the `devDependency` is `@hot-loader/react-dom@YOUR_REACT_VERSION`

Reply:
Interesting, when I ran `yarn add react-dom@npm:@hot-loader/react-dom --dev` in my local version that was encountering the error, it added `"react-dom": "npm:@hot-loader/react-dom"` verbatim into my devDependencies. Either way, your fix seems to be working so thank you for that :)